packages/phishing-controller/src/PhishingDetector.test.ts

```diff
@@ -1298,6 +1299,33 @@ describe('PhishingDetector', () => {
       );
     });
 
+    it('check the hash against c2DomainBlocklist, returning the correct result without a version with sub domains', async () => {
+      await withPhishingDetector(
+        [
+          {
+            blocklist: [],
+            fuzzylist: [],
+            c2DomainBlocklist: [
+              'a379a6f6eeafb9a55e378c118034e2751e682fab9f2d30ab13d2125586ce1947',
```

Comment:
```suggestion
              'a379a6f6eeafb9a55e378c118034e2751e682fab9f2d30ab13d2125586ce1947', // example.com
```

Just to improve readability of the test and to match the format of the other tests